bbpress/content-single-forum.php

```diff
@@ -11,7 +11,7 @@
 
 <div id="bbpress-forums">
 
-	<?php bbp_breadcrumb(); ?>
+<!--	--><?php //bbp_breadcrumb(); ?>
```

Comment:
The _intention_ is to disable part of the code but leave it visible while I'm still working on it in case I want to revert. I can easily imagine there's a better way.